## 32misc_minor_fixes.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Remove an unconditional nethack message. (# 512299)

@DPATCH@
--- screen-4.0.3.orig/attacher.c	2009-02-26 01:40:12.427283863 +0100
+++ screen-4.0.3/attacher.c	2009-02-26 01:43:02.258234277 +0100
@@ -579,7 +579,7 @@
         {
 	  fcntl(0, F_SETFL, 0);
 	  SetTTY(0, &attach_Mode);
-	  printf("\nSuddenly the Dungeon collapses!! - You die...\n");
+	  printf("\nError: Cannot find master process to attach to!\n");
 	  eexit(1);
         }
 #endif
